### PR TITLE
Add box state

### PR DESCRIPTION
**Pull request by HaGuesto**

### CODE REVIEW

**Comment by HaGuesto** [comment]
[Start to manage box state correctly #python #database #medium](https://trello.com/c/RDe3cnsb/529-start-to-manage-box-state-correctly-python-database-medium)

File: db/migrations/20200608185725_add_box_state.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                l.is_scrap
+        ');
+
+        $this->execute('
+            UPDATE 
+                stock s
+            LEFT JOIN 
+                locations l ON l.id=s.location_id
+            SET 
+                s.box_state_id = 2
```
Comment (aerinsol) [inline db/migrations/20200608185725_add_box_state.php]: @HaGuesto I think you mean SET s.box_state_id = 1 here (2 is lost, 1 is in-stock according to your earlier definition).

**Comment by aerinsol** [comment]
@HaGuesto I think you made a mistake for instock boxes. Take a look.

File: db/migrations/20200608185725_add_box_state.php (hunk 1 above)
Comment (HaGuesto) [reply db/migrations/20200608185725_add_box_state.php]: I think you commented on an outdated git diff log. I cannot reproduce anymore on which line you commented.

image

Can you take another look @aerinsol ?

**Pull request merged**